api/src/api/v1/types.ts

```diff
@@ -230,11 +230,13 @@ export interface GetMyVolunteerLogsAggregateRequest extends Hapi.Request {
   };
 }
 
+export type SyncVolunteerLogPayload = (
```

Comment:
Technically this isn't the payload, it's one element of the array. Can we title it something like:
- `SyncVolunteerLogPayloadItem`
- `SyncVolunteerLogPayloadElement`